scrapper.py

```diff
@@ -25,39 +33,94 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        for article_link in article_bs.find_all("a", class_="article__download button-icon"):
+            self.urls.append(''.join(["https://journals.kantiana.ru/", article_link.get("href")]))
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+
+        headers = {
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 '
+                          '(KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36'}
+        sleep_period = random.randrange(3, 7)
+
+        for url in self.seed_urls:
+            response = requests.get(url, headers=headers)
+            if not response.ok:
+                print("Request was unsuccessful.")
+
+            article = BeautifulSoup(response.text, features="html.parser")
+            self._extract_url(article)
+            time.sleep(sleep_period)
```

Comment:
I think that the idea was great, but do you in reality sleep for random times? Or the delay is always the same? Be attentive!